static/js/actions/index.js

```diff
@@ -1,6 +1,35 @@
-export const UPDATE_CHECKBOX = 'UPDATE_CHECKBOX';
+// @flow
+import { createAction } from 'redux-actions';
+import type { Dispatch } from 'redux';
 
-export const updateCheckbox = checked => ({
-  type: UPDATE_CHECKBOX,
-  payload: { checked }
-});
+import * as api from '../lib/api';
+import type { PaymentResponse } from '../lib/api';
+import type { Dispatcher } from '../flow/reduxTypes';
+
+export const REQUEST_PAYMENT = 'REQUEST_PAYMENT';
+export const RECEIVE_PAYMENT_SUCCESS = 'RECEIVE_PAYMENT_SUCCESS';
+export const RECEIVE_PAYMENT_FAILURE = 'RECEIVE_PAYMENT_FAILURE';
+
+export const SET_TOTAL = 'SET_TOTAL';
+
+
+export const requestPayment = createAction(REQUEST_PAYMENT);
+export const receivePaymentSuccess = createAction(RECEIVE_PAYMENT_SUCCESS);
+export const receivePaymentFailure = createAction(RECEIVE_PAYMENT_FAILURE);
```

Comment:
nit: I like to group the action type declaration and the call to `createAction` together, like:

```js
export const ACTION_TYPE = 'ACTION_TYPE';
export const actionType = createAction(ACTION_TYPE);
```